libs/dal/src/repositories/execution-details/execution-details.schema.ts

```diff
@@ -111,6 +110,12 @@ executionDetailsSchema.index({
   _environmentId: 1,
 });
 
+/*
+ * This index was created to push entries to Online Archive
+ */
+
+executionDetailsSchema.index({ createdAt: 1 });
```

Comment:
i think we don't need the extra empty line here 😎
```suggestion
executionDetailsSchema.index({ createdAt: 1 });
```

Reply:
Done